Python/ceval.c

```diff
@@ -3060,6 +3060,34 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, PyFrameObject *f, int throwflag)
             goto error;
         }
 
+        case TARGET(MAKE_CELL): {
+            if (GETLOCAL(oparg) != NULL) {
+                // A cell must have been set in PyFrame_LocalsToFast().
+                // (This is unlikely to happen.)
+                assert(PyCell_Check(GETLOCAL(oparg)));
+                DISPATCH();
+            }
+            PyObject *cell = PyCell_New(NULL);
+            if (cell == NULL) {
+                goto error;
+            }
+            /* If it is an arg then copy the arg into the cell. */
+            if (co->co_cell2arg != NULL) {
```

Comment:
Why are we doing this? Isn't the point of this PR to treat arguments as any other locals?

Reply:
I split up the main objective of faster-cpython/ideas#41 into separate branches to keep the PRs smaller.  This one just adds `MAKE_CELL` while the next one is where we actually eliminate the arg/cell duplication.  That's where we can get rid of co_cell2arg.